lib/active_utils/currency_code.rb

```diff
@@ -11,8 +11,8 @@ class CurrencyCode
       "KES", "KGS", "KHR", "KRW", "KWD", "KYD", "KZT", "LBP", "LKR", "LTL", "LVL", "MAD", "MDL", "MGA",
       "MKD", "MMK", "MNT", "MOP", "MUR", "MVR", "MXN", "MYR", "MZN", "NAD", "NGN", "NIO", "NOK", "NPR",
       "NZD", "OMR", "PEN", "PGK", "PHP", "PKR", "PLN", "PYG", "QAR", "RON", "RSD", "RUB", "RWF", "SAR",
-      "SCR", "SEK", "SGD", "STD", "SYP", "THB", "TMT", "TND", "TRY", "TTD", "TWD", "TZS", "UAH", "UGX",
-      "USD", "UYU", "VEF", "VND", "VUV", "WST", "XAF", "XCD", "XOF", "XPF", "ZAR", "ZMW", 
+      "SCR", "SEK", "SGD", "STD", "SYP", "SZL", "THB", "TMT", "TND", "TRY", "TTD", "TWD", "TZS", "UAH",
+      "UGX", "USD", "UYU", "VEF", "VND", "VUV", "WST", "XAF", "XCD", "XOF", "XPF", "ZAR", "ZMW",
```

Comment:
Instead of playing whack-a-mole with currencies each time we come across a new one, can we just add them all and be done with it? https://www.currency-iso.org/en/home/tables/table-a1.html is the reference.

The fix for the breaking build is having Travis install a new bundler it seems: https://github.com/bundler/bundler/issues/3558